frontend/src/pages/Common/UserSelection.js

```diff
@@ -54,6 +54,7 @@ const styles = theme => ({
 });
 
 function UserSelection(props) {
+  const { classes, users, addToSelection, selectedItems, handleDelete } = props;
```

Comment:
I moved props to top of func body, nobody expects to find props deep down in React component.